lib/v2/screens/app/app.dart

```diff
@@ -96,6 +97,9 @@ class _AppState extends State<App> with WidgetsBindingObserver {
           ),
           bottomNavigationBar: BlocBuilder<AppBloc, AppState>(
             builder: (context, state) {
+              if (state.showGuardianRecoveryAlert) {
```

Comment:
if there is a recovery started, and you are logged in, it means you are getting hacked. 
So the only thing we allow you to do is to cancel the recovery. 

If you are not getting hacked, and you actually want the recovery to happen, then you will have to wait 24 hours.

Reply:
okay, but this condition shouldn't it be in a listener instead of the builder?

Reply:
No, because the alert is a full page, not a dialog or a snakbar.